### ISSUE TITLE
TypeError: 'dict' object is not callable

### ISSUE THREAD

**Issue by lpierron** [tcgoetz/GarminDB#97]
**Describe the bug**
Python crashes when running `make graphs`..

**To Reproduce**
```bash
 make graphs
/usr/local/bin/python3 graphs.py --all
Traceback (most recent call last):
  File "/Users/pierron/Developpements/GARMIN/GarminDB/graphs.py", line 304, in <module>
    main(sys.argv[1:])
  File "/Users/pierron/Developpements/GARMIN/GarminDB/graphs.py", line 288, in main
    graph.graph_activity('hr', args.period, args.days)
  File "/Users/pierron/Developpements/GARMIN/GarminDB/graphs.py", line 208, in graph_activity
    graph_func(time, data, period)
  File "/Users/pierron/Developpements/GARMIN/GarminDB/graphs.py", line 176, in _graph_hr
    self.__graph_multiple(time, [rhr, inactive_hr], 'Heart Rate', period, ['RHR', 'Inactive hr'], [(30, 100), (30, 100)], self.save)
  File "/Users/pierron/Developpements/GARMIN/GarminDB/graphs.py", line 108, in __graph_multiple
    figure = plt.figure(figsize=GarminDBConfigManager.graphs('size'))
TypeError: 'dict' object is not callable
make: *** [graphs] Error 1
```

**Additional context**
- OS : Mac OS BigSur 11.2
- Python : Python 3.9.1

**Comment by j-paulus** [comment]
Quick fix: in graphs.py, lines 108 and 133, change "round brackets" to "square brackets":
        figure = plt.figure(figsize=GarminDBConfigManager.graphs('size)])
->
        figure = plt.figure(figsize=GarminDBConfigManager.graphs['size'])

**Comment by tcgoetz** [comment]
fix is on the develop branch and will be merged to master soon: d8621e7070e157a601ff12725156ddbdb051bba3
